Makefile

```diff
@@ -90,6 +93,22 @@ image: build_image
 	@mv -f deployments/image/configmap/Dockerfile.bkp deployments/image/configmap/Dockerfile
 	@rm -f ./deployments/image/configmap/${BINARY}
 
+.PHONY: admission
+admission:
+	@echo "building admission controller binary"
+	CGO_ENABLED=0 GOOS=linux GOARCH=amd64 \
+	go build -a -o=${ADMISSION_CONTROLLER_BIN_DIR}/${POD_ADMISSION_CONTROLLER_BINARY} -ldflags \
+    '-extldflags "-static" -X main.version=${VERSION} -X main.date=${DATE}' \
+    -tags netgo -installsuffix netgo \
+    ./pkg/plugin/admissioncontrollers/webhook
+
+
+.PHONY: adm_image
+adm_image: admission
+	@echo "building admission controller docker images"
+	@cp ${ADMISSION_CONTROLLER_BIN_DIR}/${POD_ADMISSION_CONTROLLER_BINARY} ./deployments/image/admission
+	docker build ./deployments/image/admission -t yunikorn/scheduler-admission-controller:${VERSION} \
```

Comment:
The slash at the end of the line should not be there.

Reply:
Removed.